hack/migrate-tf-buckets.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+# one-off script to migrate terraforms state files to the appropriate buckets
+set -x
+
+function tfstate_cp() {
```

Comment:
This function has a similar logic in Terraform. Once you change the bucket in the provider field, TF will copy the state to this new destination. An example: https://www.terraform.io/docs/cloud/migrate/index.html#step-6-run-terraform-init-to-migrate-the-workspace.

Reply:
I'll remove this script from the PR and update terraform then, once a few in-flight PRs have landed

Still using this as a test run to ensure we org admins can view the different buckets.  Still having problems copying aws resources over atm due to use of hmac keys